stage2/include/cstdint

```diff
@@ -0,0 +1,43 @@
+#ifndef _CXXSHIM_CSTDINT
+#define _CXXSHIM_CSTDINT
+
+#include <stdint.h>
+
+namespace std {
+
+using int8_t = ::int8_t;
+using int16_t = ::int16_t;
+using int32_t = ::int32_t;
+using int64_t = ::int64_t;
+
+using int_fast8_t = ::int_fast8_t;
+using int_fast16_t = ::int_fast16_t;
+using int_fast32_t = ::int_fast32_t;
+using int_fast64_t = ::int_fast64_t;
+
+using int_least8_t = ::int_least8_t;
+using int_least16_t = ::int_least16_t;
+using int_least32_t = ::int_least32_t;
+using int_least64_t = ::int_least64_t;
+
+using intmax_t = ::intmax_t;
+using intptr_t = ::intptr_t;
```

Comment:
What about `uintptr_t`?

Reply:
ah ok, yea, I forgot `uintptr_t` and  `uintmax_t`